### ISSUE TITLE
bibref parser: bug in separating year

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#15]
@gsautter taxonomists often separate articles published by the same author in the same year with XXXXa, XXXXb, etc. In the markup process, the year is separtaed from the lower case letter. The problem is, that the lower case letter then becomes  part of the titel and thus produces routinely wrong titles

![image](https://cloud.githubusercontent.com/assets/4609956/21025483/ee41fe20-bd88-11e6-9dd9-387cd0f7a9bc.png)

![image](https://cloud.githubusercontent.com/assets/4609956/21025537/167a1800-bd89-11e6-8671-bf0eeeab4533.png)

**Comment by gsautter** [comment]
I'm well aware of those index letters, and citation tagging observes them.

The nasty part about this very document is that there is no punctuation at all separating year from title - in the very most references I've seen so far, there tends to be a comma or dot, or the year is in parentheses, which acts as a natural separator. For complete lack of the latter, this one will be kind of hard to resolve automatically, I'm afraid ... which document is it, anyway? Without that, it's hard to even give it a shot.

**Comment by myrmoteras** [comment]
http://tb.plazi.org/GgServer/summary/FFB60C131215FF9FFFFAFFA0EA6DFFE9

**Comment by gsautter** [comment]
OK, truncating those leading lower case letters now if immediately preceded by a year.

I also tweaked a few patterns (in a config file, of course) to adjust our RefParse version a bit more towards journal-heavy taxonomy, away from more conference-oriented computer science.
Namely, I removed filters that prevent the year following a conference acronym (like in "TDWG 2014") to be mistaken for a volume number, and on top of that added filters that do the same for years in taxonomic authorities (like in "Aus bus (Agosti, 2014)").

Works a lot better now on this article, those few tweaks did away with all the errors in all but one reference, which has more subtle reasons for incurring an error that are _way_ harder to counter.
